### PR TITLE
Add Gelma's typos that start with "s"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -47528,9 +47544,18 @@ satistied->satisfied
...
+satisying->satisfying
 satric->satiric
 satrical->satirical
 satrically->satirically
+satsifaction->satisfaction
+satsifactorily->satisfactorily
+satsifactory->satisfactory
+satsified->satisfied
+satsifies->satisfies
+satsify->satisfy
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:47557]: Perhaps `salsify`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -47789,13 +47831,24 @@ scritps->scripts
 scritpt->script
 scritpts->scripts
 scrits->scripts
+scroe->score
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:47834]: `scrod`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -47789,13 +47831,24 @@ scritps->scripts
...
+scroed->scored, scrolled,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:47835]: `scrod` again?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -47941,8 +48017,12 @@ secoundly->secondly
 secounds->seconds
 secquence->sequence
 secratary->secretary
+secrect->secret, secrecy,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:48020]: `erect` (and verbal forms)?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -49342,6 +49449,9 @@ simulaiton->simulation
 simulaitons->simulations
 simulantaneous->simultaneous
 simulantaneously->simultaneously
+simular->similar, singular,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:49452]: Perhaps `simulator`?

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -50686,6 +50826,11 @@ spindel->spindle
 spindels->spindles
 spinlcok->spinlock
 spinock->spinlock
+splel->spell
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:50829]: Perhaps `spiel`?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -50759,11 +50907,16 @@ sqaure->square
 sqaured->squared
 sqaures->squares
 sqeuence->sequence
+sqllite->SQLite
 squashgin->squashing
 squence->sequence
 squirel->squirrel
 squirl->squirrel
 squrared->squared
+squre->square, squire,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:50916]: Also `sure`? **s** and **q** are neighbours on my keyboard.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -50759,11 +50907,16 @@ sqaure->square
...
+squred->squared
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:50917]: Also `squired` as above?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -50759,11 +50907,16 @@ sqaure->square
...
+squres->squares, squires,
+squring->squaring
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:50919]: Also `squiring` as above?

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -51040,9 +51199,12 @@ statuline->statusline
 statulines->statuslines
 statup->startup
 staturday->Saturday
+statuse->status, statuses, statues, statute,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:51202]: Also `stature`?

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -52334,6 +52514,7 @@ supspicions->suspicions
 supspicious->suspicious
 supspiciously->suspiciously
 surbert->sherbet
+surelly->surely
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:52517]: Also `surreally`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:47557]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:48020]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**